neofetch

```diff
@@ -1511,17 +1505,7 @@ get_model() {
                 fi
 
                 case $model in
-                    Mac14,15):              "MacBook Air (15-inch, M2, 2023)" ;;
-                    Mac14,14):              "Mac Studio (M2 Max, 2023)" ;;
-                    Mac14,13):              "Mac Studio (M2 Ultra, 2023)" ;;
-                    Mac14,8):               "Mac Pro (2023)" ;;
-                    Mac14,6 | Mac14,10):    "MacBook Pro (16-inch, 2023)" ;;
-                    Mac14,5 | Mac14,9):     "MacBook Pro (14-inch, 2023)" ;;
-                    Mac14,12 | Mac14,3):    "Mac mini (2023)" ;;
                     Mac14,7):               "MacBook Pro (13-inch, M2, 2022)" ;;
-                    Mac14,2):               "MacBook Air (M2, 2022)" ;;
-                    Mac13,1):               "Mac Studio (M1 Max, 2022)" ;;
-                    Mac13,2):               "Mac Studio (M1 Ultra, 2022)" ;;
                     MacBookPro18,[3-4]):    "MacBook Pro (14-inch, 2021)" ;;
```

Comment:
Please make sure to make your changes on the latest version of neofetch from this repo. Since your version is older, merging this would revert a lot of updates.